### PR TITLE
Ghactions test 202101

### PR DESCRIPTION
**Pull request by ericherman**
Actions seem to be working fine!

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: Test
+on: push
+
+defaults:
+  run:
+    shell: bash
+    # working-directory: scripts
+
+jobs:
+  job1:
```
Comment (bvhme) [inline .github/workflows/test.yml:10]: Can we give this job a more telling name like 'cibuild' or something :)

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+name: Test
+on: push
+
+defaults:
+  run:
+    shell: bash
+    # working-directory: scripts
```
Comment (bvhme) [inline .github/workflows/test.yml:7]: This can probably be removed

**Review by bvhme — APPROVED** [review]

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+name: Test
+on: push
+
```
Comment (felixfaassen) [inline .github/workflows/test.yml:3]: @ericherman we could consider also triggering the test on a pull request (on a certain branch):  
```
on: push
      pull_request:
          branches:
             - master
```

**Review by felixfaassen — COMMENTED** [review]
We could consider adding a trigger condition for the test on pull_request (see comment on the test.yml. But good also do it later. Not a blocker ;-)

**Comment by ericherman** [comment]
I'll merge this as a start and then we can add  triggering the test on a pull request separately. Maybe we would want to do this on any PR to master or develop?

**Pull request merged**